packages/trader/src/App/app.tsx

```diff
@@ -24,7 +25,19 @@ const TradeModals = Loadable({
 
 const App = ({ passthrough }: Apptypes) => {
     const root_store = initStore(passthrough.root_store, passthrough.WS);
+
     React.useEffect(() => {
+        if (process.env.RUDDERSTACK_STAGING_KEY || process.env.RUDDERSTACK_PRODUCTION_KEY) {
```

Comment:
```suggestion
        if (process.env.RUDDERSTACK_KEY) {
```

since we will be using only this env var